bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/GC.java

```diff
@@ -3887,6 +3909,7 @@ long identity() {
 
 void init(Drawable drawable, GCData data, long hDC) {
 	int foreground = data.foreground;
+	this.data = data;
```

Comment:
Why move it up?

Reply:
moved back

Reply:
As discussed in the call: leave it at the end (_i.e._ revert this change) and adapt the usages of `this.data` to avoid using it before the `init` method has finished.